themes/custom-blist/assets/css/site.css

```diff
@@ -3,6 +3,10 @@ body {
   font-family: "Inter", sans-serif;
 }
 
+.section-title:hover .section-link-icon, .section-link-icon:hover {
```

Comment:
Utilise directement les outils de tailwind, c'est fait pour 
https://v2.tailwindcss.com/docs/hover-focus-and-other-states

Reply:
+1